src/Files.Launcher/Files.Launcher.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0-windows10.0.22000.0</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
```

Comment:
@gave92 Was this intentional?

Reply:
No, I need it to compile (I'm on VS 2019) but was not supposed to be committed.